app/topics/topic-management/manage-reviews/manage-reviews.component.ts

```diff
@@ -0,0 +1,99 @@
+﻿import { Component, Input, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { ToasterService } from 'angular2-toaster';
+import { TranslateService } from 'ng2-translate';
+
+import { TopicService } from '../../shared/topic.service';
+import { User } from '../../../core/user/user.model';
+import { UserService } from '../../../core/user/user.service';
+
+@Component({
+  moduleId: module.id,
+  selector: 'hip-manage-reviews',
+  templateUrl: 'manage-reviews.component.html',
+  styleUrls: ['manage-reviews.component.css']
+})
+
+export class ManageReviewsComponent implements OnInit {
+  private isEnableStatusOption = true;
+  private isEnableSaveButton = false;
+  private reviews :string [] = [];
+  private selectedReviewOption: any;
+  translatedResponse: any;
```

Comment:
`translatedResponse` is not needed as global attribute. Use a local variable instead.